lib/dsl/poc/parser.js

```diff
@@ -25,7 +25,12 @@ class JDLParser extends Parser {
           { ALT: () => $.SUBRULE($.entityDeclaration) },
           { ALT: () => $.SUBRULE($.relationDecl) },
           { ALT: () => $.SUBRULE($.enumDecl) },
-          { ALT: () => $.SUBRULE($.dtoDecl) }
+          { ALT: () => $.SUBRULE($.dtoDecl) },
+          { ALT: () => $.SUBRULE($.pagiDecl) },
```

Comment:
can you, as much as possible, name the declaration so that `pagiDecl` is `paginationDeclaration` ?

Reply:
Sure, just copying from the existing grammar.

Reply:
maybe you meant **paginationDecl** ? because all the other rule names have a **decl** suffix?

Reply:
There is `entityDeclaration`, and I'd like to have everything nicely written like that. It's not important and another commit can be made to have that.